tests/test_biophysical/test_animal/test_cow_calf/test_beef_ration.py

```
@@ -0,0 +1,496 @@
+    original_names = set(original_attrs.keys())
+    yield
+    for name in list(RationManager.__dict__.keys()):
+        if name.startswith("__"):
+            continue
+        if isinstance(RationManager.__dict__[name], (types.FunctionType, classmethod, staticmethod)):
+            continue
+        if name not in original_names:
+            try:
+                delattr(RationManager, name)
+            except AttributeError:
+                pass
```

Ruff (0.15.20)

[warning] Use `contextlib.suppress(AttributeError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(AttributeError): ...`

(SIM105)